.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
```

Comment:
i think we already merged this file in 🤔 do you mind rebasing so that this doesn't show up in the diff?